app/models/general_content.rb

```diff
@@ -3,6 +3,7 @@ class GeneralContent < Node
 
   validates_with SectionHeritageValidator
   validates_presence_of :section
+  validates :parent, presence: true
```

Comment:
Yes I suppose it might be completely redundant, happy to remove it but it does otherwise seem like it's testing two distinct things. However I can't think of an instance with >0 ancestors being tested that would fail a parent presence test. I'll remove it.